modules/buy-button/README.md

```diff
@@ -0,0 +1,61 @@
+# BuyButton
```

Comment:
Add CSS namespaces. See on the @ClaudivanFilho PR of shelf

Reply:
👍